app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -1922,10 +1922,10 @@ public boolean isPermissionGranted(@NonNull String permission) {
     }
 
     @Override
-    public void requestPermission(String uri, @NonNull String permission, WSession.PermissionDelegate.Callback aCallback) {
+    public void requestPermission(String originator, @NonNull String permission, boolean isWebsite, WSession.PermissionDelegate.Callback aCallback) {
```

Comment:
Instead of a boolean (which is meaningless in the call sites) we should use an enumerated type which is far more descriptive.

Reply:
I have replaced this boolean with an enum.